### PR TITLE
feat: add HTML containing links to scripts

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add an HTML file containing a link to the script. After publishing this file in Github Pages, it will be used by APM.

Check https://github.com/hal-shu-sato/apm/pull/142 for details.

## Checklist:

- [x] The HTML file has been formatted by prettier.
- [ ] ~~The XML file has been formatted by prettier.~~
- [ ] ~~Updated the modification date in `mod.xml`.~~

### CODE REVIEW

File: docs/install-script.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (hal-shu-sato) [inline docs/install-script.html:2]: This page is written in Japanese.

```suggestion
<html lang="ja">
```

File: docs/install-script.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    />
+
+    <title>AviUtl Package Manager</title>
+  </head>
+  <body class="bg-light">
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ"
+      crossorigin="anonymous"
+    ></script>
```
Comment (hal-shu-sato) [inline docs/install-script.html:23]: It is recommended to place this just before `</body>`.

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
I have some points to change.

**Comment by mitosagi** [comment]
I fixed it.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for your improvement!

**Comment by hal-shu-sato** [comment]
I published it at https://hal-shu-sato.github.io/apm-data/.

**Pull request merged**